lib/second_factor/actions/base.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module SecondFactor::Actions
+  class Base
+    include Rails.application.routes.url_helpers
+    attr_reader :current_user, :guardian
+
+    def initialize(current_user, guardian)
```

Comment:
Are we able to just pass in the `guardian` instead since the guardian will always be created from the current user?